docs/articles/documentation/getting-started/index.md

```diff
@@ -0,0 +1,195 @@
+Therefore, you will usually have one test fixture per website page/route.
+
+To create a test fixture, just create a .js file in the *tests* directory and add code to the file.
+
+The following snippet illustrates basic elements of a TestCafe fixture:
+
+* fixture declaration - is added by using the the `fixture` keyword;
+* target webpage - is specified by using the `page` function;
+* test code - is placed within the `test` function.
+
+```js
+fixture `Fixture name`
+    .page('http://testcafe.devexpress.com/Example');
+
+test('Test 1 name', func t => {
+    //Test 1 code
+});
+
+test('Test 2 name', func t => {
+    //Test 2 code
+});
+```
+
+### Performing Actions on Page
+
+Every test should be capable of interacting with a page content. To perform user actions, TestCafe provides a number of methods: `click`, `hover`, `typeText`, `upload`, etc.
+They can be called in a chain.
+
+For example, the following fixture contains a simple test that types a developer name into a text editor and then clicks the Submit button.
+All test actions are implemented as functions of the test controller object `t`.  This object is used to access test run API.
+
+```js
+fixture `Getting Started - Test 1`
+    .page('http://testcafe.devexpress.com/Example');
+
+test('MyTest', async t => {
+    await t
+        .typeText('#Developer_Name', 'John Smith')
+        .click('#submit-button');
+});
+```
+
+For more information, see [Actions](../test-api/actions.md).
+
+### Assertions
+
+A functional test should also check the result of actions performed. For example, clicking the Submit button on the sample web page opens a "Thank you" page whose header addresses a user by the entered name.
+To check if the name is correct, you have to add an assertion to the test.
+
+TestCafe allows you to use any assertion library to perform verifications. Before calling assertions, you need to install the assertion library into your tested project.
+
+The following test demonstrates how to use an assertion from [Chai Assertion Library](http://chaijs.com/api/bdd/).
+Before running the test, install the assertion library: go to the tested project directory (*my-project*) and run `npm install chai`.
+
+```js
+fixture `Getting Started - Test 2`
+    .page('http://testcafe.devexpress.com/Example');
+
+test('MyTest', async t => {
+    await t
+        .typeText('#Developer_Name', 'John Smith')
+        .click('#submit-button');
+
+    expect(await t.eval( () => document.querySelector('#article-header').text )).to.equal('Thank You, John Smith!');
+};
+```
+
+To check if the actual text is equal to the expected one, the `.equal(value)` assertion method is used.
+It is called within the special `t.eval` function provided by TestCafe. This function allows you to obtain the state of the element you verify on the client side.
+For more information, see [Executing Client Code](../test-api/executing-client-code.md).
+
+### Dialog Handling
```

Comment:
We can remove it from `GettinStarted` since we need to describe some very basic concepts here. Everything else should be described in recipes + API reference